pkg/kubectl/resource_printer.go

```diff
@@ -402,13 +402,17 @@ func printPod(pod *api.Pod, w io.Writer, withNamespace bool) error {
 		name = pod.Name
 	}
 
+	phase := string(pod.Status.Phase)
+	if pod.DeletionTimestamp != nil {
+		phase = "Terminating"
```

Comment:
Ok for now. This would go into the Reason column if my overhaul proposal is implemented: https://github.com/GoogleCloudPlatform/kubernetes/issues/7843#issuecomment-104821801